.travis.yml

```diff
@@ -16,6 +16,8 @@ jobs:
       name: conode
       language: go
       go: "1.12.x"
+      install:
+        - export GO111MODULE=on
```

Comment:
It's already ~fixed by #157.

Reply:
I will not use cothority/v4 in this branch. Once this is in, my next step is to merge it with `master`, then it'll use `v4`...